requirements.txt

```diff
@@ -8,4 +8,5 @@ python-glanceclient
 python-cinderclient
 python-neutronclient
 python-keystoneclient
+python-ironicclient
```

Comment:
What is the ironic client used for in in Ursula's environment? Aren't all ironic calls happening from the target system itself?